lib/rucio/daemons/hermes/kafka/kafka_support.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
```

Comment:
If we wrote this, it should have our copyright, not theirs.

Reply:
Fixed - in this file and the other original files.